src/pages/components/text-input/style.mdx

```diff
@@ -1,13 +1,32 @@
 ---
 title: Text input
 description:
-  Text inputs enable the user to interact with and input content and data. This
-  component can be used for long and short form entries.
+  The following page documents visual specifications such as color, typography,
+  structure, and size.
```

Comment:
Is this something we're changing across all Style pages? 👀

Reply:
yes! It's a new item in our style tab template. We thought it would be nice to clarify what the style tab is about instead of duplicating the description of the component on the usage tab.